src/verify.py

```diff
@@ -0,0 +1,6 @@
+n = [1,2,3,4,5,6,7,8,9,64,81,69]
+k= int(input("Inserisci un numero: "))
+if k in n:
+    print("E' dentro la lista >:)")
+else:
+    print("Non è dentro la lista :(")
```

Comment:
Definisci una funzione, così da poter riutilizzare questo codice in altri progetti/esercizi.